library/_thread.po

```diff
@@ -60,32 +67,48 @@ msgid ""
 "terminates with an unhandled exception, a stack trace is printed and then "
 "the thread exits (but other threads continue to run)."
 msgstr ""
+"Démarrer un nouveau fils d'exécution et le renvoyer avec son identifiant. Ce "
+"fil d'exécution exécute la fonction *function* avec la liste d'arguments "
+"*args* (qui doit être un tuple). L'argument optionnel *kwargs* spécifie un "
+"dictionnaire d'arguments de mots clés. Quand la fonction renvoie, le fils "
+"d'exécution se termine silencieusement. Quand la fonction termine avec une "
+"erreur d'exception non gérée, une trace de la pile est imprimée et ensuite "
```

Comment:
```suggestion
"d'exception non gérée, une trace de la pile est affichée et ensuite "
```

Pourquoi "erreur d'exception" ?
On préfère afficher à imprimer, imprimer fait très "papier".

Reply:
Et, pourtant "imprimer" est le terme technique IT ! :p 
C'est même historique ;)